images/kamu-base-git/podman/podman-priv.yaml

```diff
@@ -0,0 +1,20 @@
+version: v1
+kind: Pod
+metadata:
+  name: podman-priv
```

Comment:
I'm confused. How all this kubernetes stuff is related to the docker image?

I don't see it being used anywhere.

Reply:
This is the contents of the `podman` folder that was copied from `kamu-base` (`./images/kamu-base/podman`)
If it's not relevant here, it looks like we need to do a revision of what we supply in `kamu-base`.

What do you think about that?

Reply:
Addendum: as part of this PR, these changes are no longer contained. But they also remain in `kamu-base`. This seems to be an echo from 2021. 

In a good way, we need to revisit the files that are packed into the `kamu-base` image.

Reply:
Looks like originally I copied these files to `kamu-base` by accident. I will clean them up on master.